From 55011a974c375adacd8178532a56b18315026a7a Mon Sep 17 00:00:00 2001
From: Giovanni Santini <giovannisantini93@yahoo.it>
Date: Wed, 15 Aug 2018 15:45:48 +0200
Subject: [PATCH] Enabling MSE (Media Source Extensions) and EME (Encrypted
 Media Extensions) MSE and EME support has been merged already within
 WebkitGTK+ and is useful for many video and audio streaming websites (one to
 mention, Youtube). This simple tweak enables MSE and EME when WebkitGTK+ is
 built with the proper flags.

---
 embed/ephy-embed-prefs.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/embed/ephy-embed-prefs.c b/embed/ephy-embed-prefs.c
index 0ce447f30..830d0b58f 100644
--- a/embed/ephy-embed-prefs.c
+++ b/embed/ephy-embed-prefs.c
@@ -432,6 +432,8 @@ ephy_embed_prefs_init (gpointer user_data)
                                                        "enable-javascript", TRUE,
                                                        "enable-dns-prefetching", TRUE,
                                                        "javascript-can-open-windows-automatically", TRUE,
+                                                       "enable-mediasource", TRUE,
+                                                       "enable-encrypted-media", TRUE,
                                                        NULL);

   for (i = 0; i < G_N_ELEMENTS (webkit_pref_entries); i++) {
--
2.18.0
